Packs/Palo_Alto_Networks_WildFire/Integrations/WildFireReports/WildFireReports.py

```diff
@@ -12,16 +11,27 @@ def __init__(self, base_url: str, verify: bool = True, proxy: bool = False, ok_c
                  token: str = None):
         super().__init__(base_url, verify, proxy, ok_codes, headers)
         self.token = token
-
+        self.agent = self.get_agent()   # Agent is different based on the platform running the integration (XSOAR/XSIAM)
         add_sensitive_log_strs(token)
 
+    @staticmethod
+    def get_agent():
+        """
+        Auto API expect the agent header to be 'xdr' when running from within XSIAM and 'xsoartim' when running from
+        within XSOAR (both on-prem and cloud).
+        """
+        platform = get_demisto_version().get('platform')
+        if platform == 'x2':
+            return 'xsoartim' if is_demisto_version_ge('8.0.0') else 'xdr'
+        return 'xsoartim'
```

Comment:
```suggestion
        if platform == 'x2' and not is_demisto_version_ge('8.0.0') :
            return 'xdr'
        return 'xsoartim'
```

Reply:
Why do you think this is better?

Reply:
actually realized this is better, to flip it 
`

if   is_demisto_version_ge('8.0.0')  or platform == 'xsoar':
	            return 'xsoartim'
return 'xdr'

`

I think this is clearer than the nested if. there's 2 reasons why to return xsoar